### ISSUE TITLE
fussy fuss

### ISSUE THREAD

**Issue by moander** [ocombe/angular-brunch-seed-no-fuss#7]
Nothing seems to be working.. Everything is outdated. I will write a pull request if I figure it out :)

**Comment by ocombe** [comment]
Yes, it's totally outdated, I don't use brunch and I haven't updated the repository.
If you want to take over I can add you as a collaborator.
